### DISCUSSION TITLE
Bug: word wrap setting doesn’t hold

### DISCUSSION THREAD

**Discussion by nguyenvukhang** [community/community#141389] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.181.0 (143346435)
Device: iPhone 15 (iOS 18.0.1)
</details>

This has been bothering me for a while now but in the iOS it's configured to have word wrap off but whenever I open a file, the lines are wrapped. I'll have to open the local settings and toggle word wrap off and on again to have it work.

**Comment by alcere** [comment]
Hi @nguyenvukhang! Thank you for reporting this issue! I've forwarded it to the iOS team, and they will investigate it soon!

**Comment by nguyenvukhang** [comment]
This seems fixed as of 1.183.1.
Thanks!

**Comment by nguyenvukhang** [comment]
I’m still on the same version, but as I use it more I see that `*.txt` files and `*.md` files still exhibit this bug (among others probably).

However, it’s fixed for `*.c`, `*.cpp`, `*.hpp`, and `*.h` (again, among others).

**Reply by alcere** [reply]
Thanks for following up, and we apologize for the trouble. I have shared your findings with the team, and we will try to look into it as soon as possible!

**Reply by alcere** [reply]
Toggling on/off the setting after opening the file should remediate the issue, as @nguyenvukhang described above.
